tests/functional/test_scoping/rrel_multifile/navigation1.model

```diff
@@ -0,0 +1,30 @@
+#include "navigation1_included.model"
+A a1 {
+    A aa1 {
+        A aaa1 {}
+        A aab1 {}
+        P1 aaa1
+        P1 aa1
+        P1 ab1
+        P1 a2
+    }
+    A ab1 {}
+}
+A R {
+    A r2 {}
+}
+
+// activate importURI (in any case, also for `GrammarMissingPlusM.tx`):
+
+R R.r2
+
```

Comment:
Hmm. We can also remove the whole comment... Without the "R" command, we could also pass the test w/o implementing the RREL provider correctly...

Reply:
Ah, ok. I *think* I've figured out what are you up to. I did a slight refactoring and comment rewording. I've moved `R` command before `A` and `P1`. Then in `R` I used a non local reference to show that the `R` command can find its referent in both cases since the `+m` for `R` rule is not removed. Then we can clearly see that `P1` can't be resolved when `+m` is removed. Does that looks ok?

Reply:
Yes. Very good. Thank you for the clarification. After all it is a border case.

Reply:
If that is OK I think this is ready to be squash merged.